README.md

```diff
@@ -0,0 +1,49 @@
+# Google Code-in Leaders
+***
```

Comment:
Can we remove all of these rulers? They look annoying.

Reply:
Yup as we are already using h1 above this, don't need this.